src/Dynamo.All.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28010.2050
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29806.167
```

Comment:
If possible, would you revert this change as well as the version change above? These are local IDE and timestamp dependent

Reply:
yes